### DISCUSSION TITLE
Select 多选模式是否需要支持当关闭下拉框后触发 onChange

### DISCUSSION THREAD

**Discussion by crazyair** [ant-design/ant-design#47869] [General]
如题

**Comment by MadCcc** [comment]
这个改造成本不小。。

**Comment by crazyair** [comment]
背景是搜索场景，使用 Form onValuesChange 发请求，如果 Select 不是关闭后触发 onChange，那选择一个数据就会发一个请求，就比较频繁。有2个方案
1. onValuesChange 做防抖
2. onChange 改成 onOpenChange，不过 onOpenChange 要改 2 点：1.参数需要加 value。2. 关闭才触发 Form 的 onValuesChange
